### PR TITLE
Adding ATAC-Seq slides

### PR DESCRIPTION
**Pull request by heylf**
I have added slides for the ATAC-Seq tutorial. It will be used for the tutorial video. Please check the content and organization of the slides. Furthermore, I did not know how to combine the slides with the hand-ons on the tutorial page. @lldelisle @bebatut @bgruening

### CODE REVIEW

File: topics/epigenetics/slides/introduction_atac.html
Comment (heylf) [reply topics/epigenetics/slides/introduction_atac.html:13]: Yeah I think I do the videos myself.

Comment (hexylena) [reply topics/epigenetics/slides/introduction_atac.html:13]: Check out some of them here! https://training.galaxyproject.org/training-material/topics/admin/videos/

They're pretty good. But in any case even if you don't set `video: true`, speaker notes can be nice for others to know what to say when they present your slides.

Comment (heylf) [reply topics/epigenetics/slides/introduction_atac.html:13]: Mmm ... sounds actually really good. I listened to different example. Ok I will add more speaker notes.

**Comment by hexylena** [comment]
Fantastic! And in case I miscommunicated earlier - it'd be great if you still want to produce a video of you giving the slides, I think people really like the "human" element, but this is a nice fallback for if that video ever becomes outdated.

**Comment by heylf** [comment]
> Fantastic! And in case I miscommunicated earlier - it'd be great if you still want to produce a video of you giving the slides, I think people really like the "human" element, but this is a nice fallback for if that video ever becomes outdated.

Oh yeah I will do. Yet, I will first record the hands-on because I want to wait for sb. to check the slides. Maybe they can be still improved.

**Comment by hexylena** [comment]
Awesome!!!

**Comment by lldelisle** [comment]
Wouah, super nice.

**Comment by lldelisle** [comment]
@heylf we have a training with @hrhotz in 1 months and a half (https://www.sib.swiss/training/course/20210428_GALXY). Maybe you should wait before doing the video of the hands-on as we may modify it...

**Comment by heylf** [comment]
> @heylf we have a training with @hrhotz in 1 months and a half (https://www.sib.swiss/training/course/20210428_GALXY). Maybe you should wait before doing the video of the hands-on as we may modify it...

Ah ok. Yeah let me know, if you did any changes. Then I willl wait to do the recording. Can I maybe join the call? Maybe you say something (some details) which could be nice for the user to know.

**Comment by hrhotz** [comment]
sure, any "extra hands" are welcome

**Comment by hexylena** [comment]
@heylf is this ready to go or were you waiting on joining that meeting?

**Comment by lldelisle** [comment]
I would like to do review before it is merged...

File: topics/epigenetics/slides/introduction_atac.html
Comment (lldelisle) [inline topics/epigenetics/slides/introduction_atac.html:44]: I am not sure I understand this. There is a sequence bias but the Tn5 can cut everywhere...

File: topics/epigenetics/slides/introduction_atac.html
Comment (lldelisle) [inline topics/epigenetics/slides/introduction_atac.html:46]: They are not filtered out. If the length is too big (>800bp) it is not correctly amplified by PCR and/or not efficiently sequenced by illumina sequencers but we for sure keep the pairs with 1 or 2 nucleosomes inside. That's why it is so important to build coverage around the insertion sites and not fully between the mates of the pairs: if the pairs are larger than 170bp you don't know if there was a nucleosome between them or if everyting was 'naked' but you know that where the Tn5 cuts there was no nucleaosome.

**Comment by hexylena** [comment]
Absolutely, that's fantastic to hear :) Just let us know when you're ready!

File: topics/epigenetics/slides/introduction_atac.html
Comment (lldelisle) [inline topics/epigenetics/slides/introduction_atac.html:53]: I don't think they are the usual controls. For me the controls are purified DNA (no more nucleosome) treated with Tn5.

File: topics/epigenetics/slides/introduction_atac.html
Comment (lldelisle) [inline topics/epigenetics/slides/introduction_atac.html:54]: I think it is usually paired-end because you can select pairs which would be separated less than 150-170bp and then you are sure there is no nucleosome in between but most of application when you just use the 5' of all reads you don't really need paired-end.

File: topics/epigenetics/slides/introduction_atac.html
Comment (lldelisle) [inline topics/epigenetics/slides/introduction_atac.html:67]: This sentence is strange to me. I agree with the following 2.

File: topics/epigenetics/slides/introduction_atac.html
Comment (lldelisle) [inline topics/epigenetics/slides/introduction_atac.html:92]: I would put this at the end.

File: topics/epigenetics/slides/introduction_atac.html
Comment (lldelisle) [inline topics/epigenetics/slides/introduction_atac.html:97]: Especially because there is no nucleosome on the MT... So it makes this part of the genome very sensitive to Tn5.

File: topics/epigenetics/slides/introduction_atac.html
Comment (lldelisle) [inline topics/epigenetics/slides/introduction_atac.html:99]: - intersted -> interested
- I used to use local alignment to go faster and do not trim adapters. I think it is more clean to do 1. trim adapter 2. end-to-end alignment but I would not say that it is that important.

File: topics/epigenetics/slides/introduction_atac.html
Comment (lldelisle) [inline topics/epigenetics/slides/introduction_atac.html:107]: Macs2 is not taking into consideration the 9bp insertion. So I would say: "You would prefer".

**Review by lldelisle — COMMENTED** [review]
Hi,
Thank you very much for wrapping these slides.
I have to warn you that I am very direct when I do reviews, please do not take it personally...
If you want we can discuss some points.

**Comment by heylf** [comment]
> Hi,
> Thank you very much for wrapping these slides.
> I have to warn you that I am very direct when I do reviews, please do not take it personally...
> If you want we can discuss some points.

Oh dont worry. I am used to critic. The shell became thick because of paper revision ^^

File: topics/epigenetics/slides/introduction_atac.html
Comment (heylf) [reply topics/epigenetics/slides/introduction_atac.html:44]: Maybe I missunderstood this paper then: ATAC-seq: A Method for Assaying Chromatin Accessibility 
Genome-Wide, Jason Buenrostro [2016]. It states:  Tn5 is a prokaryotic transposase, which endogenously functions through the “cut and paste” mechanism, requiring sequence-specific excision of a locus containing 19 base-pair inverted repeats.

What does this sentence then mean?

File: topics/epigenetics/slides/introduction_atac.html
Comment (heylf) [reply topics/epigenetics/slides/introduction_atac.html:46]: check

File: topics/epigenetics/slides/introduction_atac.html
Comment (heylf) [reply topics/epigenetics/slides/introduction_atac.html:53]: The wording for the control I got from these guidelines: https://informatics.fas.harvard.edu/atac-seq-guidelines.html

File: topics/epigenetics/slides/introduction_atac.html
Comment (heylf) [reply topics/epigenetics/slides/introduction_atac.html:54]: check

File: topics/epigenetics/slides/introduction_atac.html
Comment (heylf) [reply topics/epigenetics/slides/introduction_atac.html:67]: check

File: topics/epigenetics/slides/introduction_atac.html
Comment (heylf) [reply topics/epigenetics/slides/introduction_atac.html:92]: You mean the sentences or the entire slide?

File: topics/epigenetics/slides/introduction_atac.html
Comment (heylf) [reply topics/epigenetics/slides/introduction_atac.html:97]: check

File: topics/epigenetics/slides/introduction_atac.html
Comment (heylf) [reply topics/epigenetics/slides/introduction_atac.html:99]: check

File: topics/epigenetics/slides/introduction_atac.html
Comment (heylf) [reply topics/epigenetics/slides/introduction_atac.html:107]: check

File: topics/epigenetics/slides/introduction_atac.html
Comment (lldelisle) [reply topics/epigenetics/slides/introduction_atac.html:44]: The original transposase Tn5 indeed only recognize the 19bp inverted repeat (https://onlinelibrary.wiley.com/doi/full/10.1046/j.1365-2958.2003.03382.x) but the hyperactive Tn5 is less restrictive.

File: topics/epigenetics/slides/introduction_atac.html
Comment (lldelisle) [reply topics/epigenetics/slides/introduction_atac.html:56]: Ah... For me this would not be a good control as there is also a bias during sonication for open chromatin...

File: topics/epigenetics/slides/introduction_atac.html
Comment (lldelisle) [reply topics/epigenetics/slides/introduction_atac.html:96]: Perfect like this

File: topics/epigenetics/slides/introduction_atac.html
Comment (heylf) [reply topics/epigenetics/slides/introduction_atac.html:44]: check

File: topics/epigenetics/slides/introduction_atac.html
Comment (heylf) [reply topics/epigenetics/slides/introduction_atac.html]: check

**Comment by heylf** [comment]
@lldelisle I revised the slides.

File: topics/epigenetics/slides/introduction_atac.html
Comment (lldelisle) [inline topics/epigenetics/slides/introduction_atac.html:54]:
```suggestion
- You may also have a control. A control could be purified DNA, which has no more nucleosomes, treated with Tn5. It is sequenced along with the ATAC sample.
```

**Comment by lldelisle** [comment]
I made a final suggestion and for me this is good.

**Comment by heylf** [comment]
I included the final suggestion.

**Review by lldelisle — APPROVED** [review]
Nice

**Comment by shiltemann** [comment]
Thanks for the slides @heylf and thanks for the review @lldelisle! 

@lldelisle since you have agreed to be a topic maintainer for epigenetics topic (thanks!!), you can merge this one if you would like? :) To do that:
1. Approve the PR (under "files changed" you should see a green "Review changes" button, and select "approve" here)
2. If all tests pass and there is at least one approving review, a PR can be merged

You should feel empowered to review and merge anything in the epigenetics topic. And if you would like a second pair of eyes reviewing for the technical side of things, feel free to ping me/others

**Review by lldelisle — APPROVED** [review]
I reapprove with my new 'authority' ;-)

**Comment by lldelisle** [comment]
Euh... Seems that it did not work...

**Review by shiltemann — APPROVED** [review]

**Pull request merged**
